cloud-server-auth/main.go

```diff
@@ -100,6 +100,10 @@ func isAuthorized(url *url.URL) bool {
 		"/api/installation",
 		"api/cluster_installation",
 		"/api/cluster_installation",
+		"api/webhooks",
+		"/api/webhooks",
```

Comment:
I'm not sure with and without the leading `/` is necessary but the existing ones have both so I put both